### PR TITLE
Add cat pics

### LINKED ISSUE #1: not enough cats
more cats

### PR DESCRIPTION
**Pull request by jamieliu386**
Fixes: #1

### CODE REVIEW

**Comment by jamieliu386** [comment]
@ktan17 please review :)

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -13,6 +16,27 @@ const Typography = withStyles({
 })(MuiTypography);
 
 export default function Home() {
+	const data = useStaticQuery(graphql`
+		{
+			catImages: allFile(filter: {relativePath: {glob:"cats/*"}}) {
```
Comment (ktan17) [inline src/components/Home/Home.js]: space after glob:

also prefer placing string literals in constants but up to you

**Review by ktan17 — APPROVED** [review]
a very exciting PR that is sure to exponentially increase this project's success

File: src/components/Home/Home.js (hunk 1 above)
Comment (TimothyGu) [reply src/components/Home/Home.js]: You can't put string literal in constant here due to the way gatsby (and also tagged template) works. However, you _can_ consider using a page query – doesn't make a difference though.

**Review by TimothyGu — APPROVED** [review]
cats are cute. I like cats

**Pull request merged**